pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[project]
+name = "pan3d"
+version = "1.0.0"
+description = "Utility package for processing and visualizing 3D datasets"
+authors = [
+    {name = "Kitware Inc."},
+]
+dependencies = [
+    "dask",
+    "fsspec",
+    "matplotlib",
+    "netCDF4",
+    "pyvista",
+    "pyvista-xarray",
+    "trame",
+    "trame-vtk",
+    "trame-vuetify",
+    "vtk",
+    "xarray",
+    "zarr",
+]
+requires-python = ">=3"
```

Comment:
3.7?

Reply:
Changed in [bffff73](https://github.com/Kitware/pan3d/pull/39/commits/bffff73681a051707ed5a100e8e86bebec4a030b)